### PR TITLE
API-24681-526-schema-draft

### PR DESCRIPTION
**Pull request by stiehlrod**
## Summary

- To help unblock VA.gov with their discovery work for migrating to the Claims API v2 526 services, we want to publish a proposed schema draft in our lower environments. This will act as a reference point for what we are planning to support in v2. Changes are expected as we continue development. 
- team Dash

## Related issue(s)
- [API-24681](https://vajira.max.gov/browse/API-24681)

## Testing done

- rswag rake tests/rspec
- rswag localhost

## What areas of the site does it impact?
modules/claims_api/config/schemas/v2/526.json

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

File: modules/claims_api/lib/claims_api/form_schemas.rb [hunk 1]
```diff
@@ -4,8 +4,13 @@
 
 module ClaimsApi
   class FormSchemas < JsonSchema::FormSchemas
+    def initialize(schema_version: 'v1')
+      # super
```
Comment (stiehlrod) [inline modules/claims_api/lib/claims_api/form_schemas.rb:8]: @FonzMP Rubocop wants 'super' here, but that breaks SO many things

Comment (FonzMP) [reply modules/claims_api/lib/claims_api/form_schemas.rb:8]: interesting. How is the Appeals API version bypassing this?

**Comment by FonzMP** [comment]
Just want to check with Kayla tomorrow morning to see if there's any reason we haven't been doing the schema endpoint for v2. Or any reason we wouldn't want to. Other than that, I'm good with it.

File: modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json [hunk 2]
```diff
@@ -1380,81 +1380,14 @@
...
-          "401": {
-            "description": "Unauthorized"
-          }
-        }
-      }
-    },
-    "/veterans/{veteranId}/526/validate": {
-      "post": {
-        "summary": "Validates a 526 claim form submission.",
+      "get": {
```
Comment (FonzMP) [inline modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json:1383]: @stiehlrod - yeah, so this will display the schema endpoint.

**Pull request merged**